config/overrides.go

```diff
@@ -361,3 +363,84 @@ func NamePrefixRemoval() tjconfig.ResourceOption {
 		r.ExternalName.OmittedFields = append(r.ExternalName.OmittedFields, "name_prefix")
 	}
 }
+
+// KnownReferencers adds referencers for fields that are known and common among
+// more than a few resources.
+func KnownReferencers() tjconfig.ResourceOption { //nolint:gocyclo
```

Comment:
Have you considered the `v1alpha2` bumps in this PR?

Reply:
I just rebased and fixed the paths to include v1alpha2